frontend/src/metabase/components/MetadataInfo/Popover/Popover.tsx

```diff
@@ -57,8 +65,10 @@ export function Popover({
     <HoverCard
       position={position}
       disabled={disabled}
-      openDelay={group.shouldDelay ? delay[0] : 0}
-      closeDelay={group.shouldDelay ? delay[1] : POPOVER_CLOSE_DELAY}
+      openDelay={
+        group.shouldDelay ? POPOVER_SLOW_OPEN_DELAY : POPOVER_FAST_OPEN_DELAY
```

Comment:
I don't think we want to make this change globally.
The bug report was specifically about column info popover in table viz headers.

We also don't want the long delay for column info popover in notebook editor ([image](https://github.com/user-attachments/assets/3353cd1b-9f0e-406f-a1c5-a1e0892a1868)).

Reply:
I personally don't think there's a need to keep separate timings for the other info icons, but I'll let @mazameli or @mngr speak to that. I think the same arguments could be made for the other info icons accross the site.

FWIW, they currently have all the same timeout: 250ms. This PR increases that to 500ms.

Reply:
@AlessioLaiso @ranquild could you guys please test it
I also don't think we should change the global delay settings. So if 250 is enough, then let's apply global settings to the column popover. If it's not enough, let's have them separate

Reply:
Okay, I've limited the increased delay to only happen on the column header popovers.